testing-rspec/acceptance_test_spec.rb

```diff
@@ -6,12 +6,12 @@
   it "signs up the user with valid details" do
     visit sign_up_path
 
-    within "#sign_up" do
+    within_fieldset "Sign up" do
       fill_in "Email", with: "user@example.com"
       fill_in "Password", with: "Examp1ePa$$"
       click_button "Sign up"
     end
 
-    expect(page).to have_content("Sign out")
+    expect(page).to have_button("Sign out")
```

Comment:
@cpjmcquillan this commit replaces the `have_content` matcher with a `have_button` matcher.

Since the sample code block is already making assumptions about the hypothetical content of the response, making the matcher _more_ specific didn't feel like a risk.

Did you have something else in mind when you introduced this code sample? Does this feel like an improvement?

Reply:
I didn't originally introduce the code sample, but I changed the formatting and some of the language used to describe the tests in #654. 

I think the change you've made is helpful, though. I can't imagine a real scenario where I would expect this test to render a page with "Sign out" content that _wasn't_ a button.